src/helpers/discord_utils.ts

```diff
@@ -131,8 +131,8 @@ export async function sendEndOfRoundMessage(messageContext: MessageContext, scor
         text: "",
     };
 
-    if (gameRound.songAliases.length > 0) {
-        footer.text += `Aliases: ${Array.from(gameRound.songAliases).join(", ")}\n`;
+    if (gameRound.acceptedSongAnswers.length > 1) {
+        footer.text += `Aliases: ${Array.from(gameRound.acceptedSongAnswers).join(", ")}\n`;
```

Comment:
Will we be showing the song name here too then (when another alias exists)?

Reply:
Yes but I don't think that's a major issue, being able to see every accepted answer